src/_gettsim/transfers/arbeitsl_geld_2/arbeitsl_geld_2.py

```diff
@@ -192,12 +192,23 @@ def arbeitsl_geld_2_kindersatz_m_hh_ab_2011(
     float with the support of children since year 2011
 
     """
+
     # Sum payments for each age group
-    out = (
-        arbeitsl_geld_2_params["regelsatz"][6] * anz_kinder_bis_6_hh
-        + arbeitsl_geld_2_params["regelsatz"][5] * anz_kinder_ab_7_bis_13_hh
-        + arbeitsl_geld_2_params["regelsatz"][4] * anz_kinder_ab_14_bis_24_hh
-    )
+    if "kindersofortzuschlag_alg" in arbeitsl_geld_2_params:
```

Comment:
I think easier and with less repetition would be, if we added the end of the function:

```
    if "kindersofortzuschlag_alg" in arbeitsl_geld_2_params:
        kindersofortzuschl = arbeitsl_geld_2_params["kindersofortzuschlag_alg"]
        out += kindersofortzuschl  * ( anz_kinder_bis_6_hh + anz_kinder_ab_7_bis_13_hh + anz_kinder_ab_14_bis_24_hh )